plutus-use-cases/test/Spec/renderGuess.txt

```diff
@@ -170,60 +170,60 @@ Balances Carried Forward:
   Value:
     Ada:      Lovelace:  100000000
 
-  Script: a1229b3c233b276767017e5f0682e323b695808699bc81d3aaf8695b3674b515
+  Script: d5c7bbcfef3f2bffd9b350bea48fa11c8f127ee0bbabe43bee14466f6b1e30c2
   Value:
     Ada:      Lovelace:  8
 
 ==== Slot #2, Tx #0 ====
-TxId:       8f7fa2645c2c43534f64b4672b3d8a28eebd7e91de2d1794db08913548f1a89e
-Fee:        Ada:      Lovelace:  20957
-Forge:      d1970901878f8ae6aa6085a561b6fe3d4e7d98a8c032ac9b114177d553e5336a:  guess:    1
+TxId:       0bae995521f9d4c183c4cdda5f9ec809e0dfd7f50bad4002d66313321a362057
+Fee:        Ada:      Lovelace:  19731
```

Comment:
@michaelpj can these values differ?

Reply:
Probably they don't, but rather the changes in the hashes has led to an ordering change.

Reply:
Hm, no, they have changed. Maybe we're now including a fee in the emulator based on tx size? That would be cool: evidence that we're making things smaller!

Reply:
Yes, the changes I have seen are:
- the fees are decreasing, and
- the remaining balance increasing.